CLDR-15197 update minimal pair PathDescriptions.md (#5836)
unicode-org/cldr: tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/PathDescriptions.md

    All other brackets must be escaped with backslash if they are to be interpreted literally rather than as regex.
    A regex may use variables starting with `%` that are defined in the `VARIABLES` section.
    Variable substitution is performed automatically.
	Note that %anyAttribute has a built-in group, so %anyAttributecounts as the first parenthesized group.
3. The final section is the markdown body, continuing up to but not including the next line beginning with `#`.
   Please keep all URLs in [References](#references), which is copied to every markdown fragment.
   This way we can share URLs and use more natural sounding links.

>    Group replacement is performed here, so for example `{0}` will get the entire XPath, `{1}` will be the first parenthesized capture group, etc. with these exceptions:

Reviewer: ```suggestion
   Group replacement is performed here, so for example `{'0'}` will get the entire XPath, `{'1'}` will be the first parenthesized capture group, etc. with these exceptions:
```

Author: no, this part's for people reading *this file in GitHub or in their source editor* to find. It doesn't show up anywhere.

>    - in the ROOT cases (first section), `{0}` is always replaced with the code

Reviewer: ```suggestion
   - in the ROOT cases (first section), `{'0'}` is always replaced with the code
```

   - in `exemplarCity`, `{0}` is always replaced with the English region name

>    - if you want a literal `{0}` in the Markdown (and who wouldn’t?), use `'{'0'}'`, `'{'1'}'` etc.

Author: this has been bothering me for a while. but i didn't investigate it, until now.

Author: and i Got it wrong, hold on

>    - in `exemplarCity`, `{0}` is always replaced with the English region name

Reviewer: ```suggestion
   - in `exemplarCity`, `{'0'}` is always replaced with the English region name
```

>    Also note, if you want a literal `{0}` in the Markdown (and who wouldn’t?), use `'{'0'}'`, `'{'1'}'` etc.

Reviewer: ```suggestion
   Also note, if you want a literal `{'0'}` in the Markdown (and who wouldn’t?), use `'{'0'}'`, `'{'1'}'` etc.
```
